feature-libs/order/components/order-details/order-detail-billing/order-detail-billing.component.html

```diff
@@ -1,15 +1,40 @@
-<div class="cx-order-items" *ngIf="order$ | async as order">
-  <div class="cx-review-summary" *ngIf="order.paymentInfo as paymentDetails">
-    <div class="cx-review-summary-card cx-review-summary-payment-card">
-      <cx-card
-        [content]="getPaymentMethodCard(paymentDetails) | async"
-      ></cx-card>
-    </div>
-
-    <div class="cx-review-summary-card cx-review-summary-payment-card">
-      <cx-card
-        [content]="getBillingAddressCard(paymentDetails) | async"
-      ></cx-card>
+<div *ngIf="true">
+  <div class="cx-order-items" *ngIf="order$ | async as order">
+    <div class="cx-review-summary" *ngIf="order.paymentInfo as paymentDetails">
+      <ng-container *cxFeatureLevel="'!6.5'">
+        <div class="cx-review-summary-card cx-review-summary-payment-card">
+          <cx-card
+            [content]="getPaymentMethodCard(paymentDetails) | async"
+          ></cx-card>
+        </div>
+      </ng-container>
+      <ng-container *cxFeatureLevel="'6.5'">
+        <div
+          class="cx-review-summary-card cx-review-summary-payment-card"
+          *ngIf="isPaymentInfoCardFull(paymentDetails)"
+        >
+          <cx-card
+            [content]="getPaymentMethodCard(paymentDetails) | async"
+          ></cx-card>
+        </div>
+      </ng-container>
+      <ng-container *cxFeatureLevel="'!6.5'">
+        <div class="cx-review-summary-card cx-review-summary-payment-card">
+          <cx-card
+            [content]="getBillingAddressCard(paymentDetails) | async"
+          ></cx-card>
+        </div>
+      </ng-container>
+      <ng-container *cxFeatureLevel="'6.5'">
```

Comment:
I think we reached a CF date, so all version numbers and target branch should be changed into 6.6.